src/js/components/homepage/features/PaneFeature.jsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * PaneFeature.jsx
+ * Created by David Trinh 4/19/18
+ */
+
+import React from 'react';
+
+const PaneFeature = () => (
+    <div className="feature-pane">
+        <div className="feature-pane__wrapper">
+            <h2 className="feature-pane__title"><span>FEATURED CONTENT</span></h2>
+            <div className="feature-pane__content-wrapper">
+                <div className="feature-pane__content">
+                    <h2 className="feature-pane__content-title">Discover the Data Lab</h2>
+                    <div className="feature-pane_content-text">
+                        <p>Data visualizations to help you understand government spending. See what our data can do.</p>
```

Comment:
Any reason that the `<p>` tags are wrapped inside a `<div>` tag rather than applying the class directly to the `<p>`?

Reply:
I did it because I wanted to apply a margin bottom to the text, but I don't think margin bottom works with inline elements. I can just add margin top to the button wrapper instead.